knative-operator/cmd/manager/main.go

```diff
@@ -75,7 +69,6 @@ func main() {
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
 		Namespace:          "", // The serverless operator always watches all namespaces.
-		MapperProvider:     restmapper.NewDynamicRESTMapper,
```

Comment:
This is now controller-runtime's default rest mapper. See https://github.com/kubernetes-sigs/controller-runtime/blob/ac380d61764a160b32946e606b0c9ecd2834e3e8/pkg/manager/manager.go#L411-L415